src/routers/index.js

```diff
@@ -1,9 +1,27 @@
 import { Router } from 'express';
+import { AddArticles, UpdateArticle, DeleteArticle } from '../controllers/article';
 
 const router = Router();
 
 router.get('/', (req, res) => res.status(200).json({
   message: 'Welcome to the Authors Haven API',
 }));
 
+/**
+ * Resource handling articles
+ * @name /api/v1/articles/:id?
+ * @function
+ * @memberof module:Express.Router
+ * @inner
+ * @param {function} AddArticles - Express path
+ * @param {function} DeleteArticle - Express path
+ * @param {function} UpdateArticle - Express path
+ * @returns Response Object
+ */
+router
+  .route('/api/v1/articles/:id?')
```

Comment:
You might want to add the base route to the server file.

Reply:
@Temmyogunbo Noted